spec/helpers/plantings_helper_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe PlantingsHelper do
+  describe "display_planting" do
+
+    before(:each) do
+      @member = FactoryGirl.build(:member,
+        :login_name => 'crop_lady'
+      )
+    end
+
+    it "neither quantity nor planted from provided" do
```

Comment:
I don't think we've ever talked about this before, but in situations where the "it" syntax doesn't make grammatical sense, should be using the alternative `specify` syntax?